cg/constants/demultiplexing.py

```diff
@@ -30,6 +30,7 @@ class DemultiplexingDirsAndFiles(StrEnum):
     CG_FILE_MANIFEST: str = "file_manifest.tsv"
     INTER_OP: str = "InterOp"
     RUN_COMPLETION_STATUS: str = "RunCompletionStatus.xml"
+    DEMUX_VERSION_FILE: str = "demux_version.txt"
```

Comment:
Is it `demux_version.txt` or `dragen_version.txt` as is defined above?

Reply:
Fixed.